packages/config/config/devices/0x0208/hkzw-scn04.json

```diff
@@ -37,6 +37,25 @@
 					"value": 1
 				}
 			]
+		},
+		"255": {
+			"label": "Reset/Remove",
+			"description": "Reset the sensor or remove from the Z-Wave network.",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 2147483647,
+			"defaultValue": 1,
```

Comment:
There should probably be an option "Idle" for the default value of 1

Reply:
I mimicked one of the other similar devices from this same manufacturer, where the same '255' parameter is used. I will confirm if the "Idle" is missing in this and the one I copied from.